qa/ccs-unavailable-clusters/src/test/java/org/elasticsearch/search/CrossClusterSearchUnavailableClusterIT.java

```diff
@@ -135,7 +135,7 @@ public void testSearchSkipUnavailable() throws IOException {
 
             for (int i = 0; i < 10; i++) {
                 restHighLevelClient.index(
-                        new IndexRequest("index", "doc", String.valueOf(i)).source("field", "value"), RequestOptions.DEFAULT);
+                        new IndexRequest("index", "_doc", String.valueOf(i)).source("field", "value"), RequestOptions.DEFAULT);
```

Comment:
For my knowledge, why didn't we remove the type altogether, as in `new IndexRequest("index").id(...))`?